cdcsdk-server/cdcsdk-server-core/src/main/java/com/yugabyte/cdcsdk/server/ServerApp.java

```diff
@@ -122,9 +123,14 @@ public class ServerApp {
     private List<DebeziumEngine<?>> engines = new ArrayList<>();
     private final Properties props = new Properties();
 
+    @ConfigProperty(name = "quarkus.application.version", defaultValue = "not-set")
+    String version;
+
     @SuppressWarnings("unchecked")
     @PostConstruct
     public void start() {
+        LOGGER.info("Version {}", version);
```

Comment:
Can you please add more description to this log statement? Something along the lines of `ServerApp version` - makes it easier to understand.